resource-timing/resource_initiator_types.html

```diff
@@ -65,6 +66,12 @@
     step_timeout(check_finished, 100);
 }
 
+function on_all_resources_loaded() {
+    all_resources_loaded = true;
+
+    step_timeout(check_finished, 100);
```

Comment:
Can the timeout be a bit smaller? It seems that it should be ready shortly after it has ran this callback.

Reply:
fixed to 10ms